src/databaseengine.h

```diff
@@ -0,0 +1,35 @@
+#ifndef DATABASEENGINE_H
+#define DATABASEENGINE_H
+
+#include <QObject>
+#include <QSqlDatabase>
+
+class DataBaseEngine: public QObject
+{
+    Q_OBJECT
+public:
+    explicit DataBaseEngine(QObject *parent = nullptr);
+
+
+    Q_INVOKABLE void connectToDataBase();
+
+public slots:
+    // Добавление записей в таблицу Логинов
+    bool insertIntoTableLog(const QString &login, const QString &hashed, const QString &email);
+
+    //    bool removeRecord(const int id); // Удаление записи из таблицы по её id
+
+private:
+    QString Direction;
```

Comment:
поля класса следует именовать с маленькой буквы и добавлять префикс m_,
```suggestion
    QString m_direction;
```